brozzler/cli.py

```diff
@@ -544,12 +544,25 @@ def dump_state(signum, frame):
         finally:
             signal.signal(signal.SIGQUIT, dump_state)
 
+    def get_skip_av_seeds():
+        skip_av_seeds_file = "/opt/local/brozzler/skip_av_seeds.txt"
+        try:
+            with open(skip_av_seeds_file) as skips:
+                skip_av_seeds = {x for x in skips.readlines()}
```

Comment:
Could you explain what the expected data structure is for this value? You're using `readlines` on a text file, which should return a list of lines, but you're trying to using a dictionary comprehension on the resulting list.

This would be clearer if you defined the expected input file format and the expected structure that should result in this variable. (Leaving the details of that up to you, but would be happy to assist.)

Reply:
OK, I checked this, and I see that it's a set comprehension. You could just skip the comprehension and cast this to a set: `set(skips.readlines())`.

Reply:
thanks, updated